js/sidenav.js

```diff
@@ -394,6 +396,7 @@
     _createDragTarget() {
       let dragTarget = document.createElement('div');
       dragTarget.classList.add('drag-target');
+      dragTarget.style.width = this.options.dragTargetWidth;
```

Comment:
hmm if we set the width via `style` attribute, will this line from SCSS necessary?

https://github.com/materializecss/materialize/blob/7bf14e073402dcc620235758e248dd83acfe0844/sass/components/_sidenav.scss#L152

Reply:
Certainly, there are no problem to delete the line. Thanks!